src/sentry/static/sentry/app/views/onboarding/createSampleEventButton.tsx

```diff
@@ -104,22 +107,36 @@ class CreateSampleEventButton extends React.Component<Props, State> {
 
     // Wait for the event to be fully processed and available on the group
     // before redirecting.
-    const eventCreated = await latestEventAvailable(api, eventData.groupID);
+    const {eventCreated, retries} = await latestEventAvailable(api, eventData.groupID);
     clearIndicators();
     this.setState({creating: false});
 
+    trackAnalyticsEvent({
+      eventKey: `sample_event.${eventCreated ? 'created' : 'failed'}`,
+      eventName: `Sample Event ${eventCreated ? 'Created' : 'Failed'}`,
+      organization_id: organization.id,
+      project_id: project.id,
+      platform: project.platform || '',
+      interval: EVENT_POLL_INTERVAL,
+      source,
+      retries,
+    });
+
     if (!eventCreated) {
       addErrorMessage(t('Failed to load sample event'));
+
+      Sentry.withScope(scope => {
+        scope.setTag('groupID', eventData.groupID);
+        scope.setTag('platform', project.platform || '');
+        scope.setTag('interval', EVENT_POLL_INTERVAL.toString());
+        scope.setTag('retries', retries.toString());
```

Comment:
Would end to end time of `latestEventAvailable` be useful to track here? Only having interval and retries does not allow us to infer that since it excludes the time taken for each request